test/testgrid/testgrid.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// testgrid.go provides methods to perform action on testgrid.
+
+package testgrid
+
+import (
+	"log"
+	"os"
+	"path"
+
+	"knative.dev/pkg/test/helpers"
+	"knative.dev/pkg/test/junit"
+	"knative.dev/pkg/test/prow"
+)
+
+const (
+	filePrefix = "junit_"
+	extension  = ".xml"
+)
+
```

Comment:
I see that you removed some of the stuff, like the jobname : tabname map, are they intended to be kept in test-infra? 

https://github.com/knative/test-infra/blob/0042e9ca752c8339c3c245e1d2c76b891a144b7e/pkg/testgrid/testgrid.go#L40

Reply:
Yes, they should still be kept in test-infra as they are tied to the testgrid.yaml file that lives in test-infra repo.